src/softDeleteGame.php

```diff
@@ -0,0 +1,12 @@
+<?php
+require_once 'DB.php';
+$id = $_POST['gameId'];
+
+$query = $db->prepare("
+UPDATE `games` SET `games`.`deleted` = 1 WHERE `games`.`id` = :id;
+");
+
+$query->bindParam(':id', $id, PDO::PARAM_INT);
+$query->execute();
```

Comment:
This query should now go into your GameModel. When you do that, you should put return before $query->execute() - that will tell you whether or not the game was deleted. If the delete fails, you should display a very basic message on the index page